components/bio-formats-tools/src/loci/formats/tools/TiffComment.java

```diff
@@ -106,9 +111,22 @@ else if (newComment.equals("-")) {
           }
         }
       }
-      else System.out.println("Warning: unknown flag: " + args[i]);
+      else if (args[i].equals(CommandLineTools.VERSION)) {
+        CommandLineTools.printVersion();
```

Comment:
Comparing the help (and the implementation) with `ImageInfo` and `ImageConverter`, should this print the version of the library and exit immediately to mirror the other utilities?
Or is it useful to be able to support `./tiffcomment -version file.tiff`?

Reply:
b8d3ae1 should now exit after printing the version, for consistency with `showinf`/`bfconvert`.